Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/AboutDialogFragment.kt

```diff
@@ -0,0 +1,92 @@
+package org.dolphinemu.dolphinemu.fragments
+
+import android.os.Bundle
+import android.text.method.LinkMovementMethod
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.dolphinemu.dolphinemu.BuildConfig
+import org.dolphinemu.dolphinemu.R
+import org.dolphinemu.dolphinemu.databinding.DialogAboutBinding
+import org.dolphinemu.dolphinemu.databinding.DialogAboutTvBinding
+
+class AboutDialogFragment : BottomSheetDialogFragment() {
+    private var _mBindingMobile: DialogAboutBinding? = null
+    private var _mBindingTv: DialogAboutTvBinding? = null
+
+    private val mBindingMobile get() = _mBindingMobile!!
+    private val mBindingTv get() = _mBindingTv!!
```

Comment:
What's the purpose of having a private getter?

If you are going to have a getter for this, please remove the m prefix.

Reply:
This is mostly to avoid putting non-null assersions everywhere ("!!." or "?."). But wouldn't it make sense to keep the m prefix for mBindingMobile/mBindingTv?

Reply:
> This is mostly to avoid putting non-null assertions everywhere ("!!." or "?.").

I suppose that makes sense.

> But wouldn't it make sense to keep the m prefix for mBindingMobile/mBindingTv?

No, why would it? It's not a member variable.

Also, a broader question: Should we be using the m prefix at all when writing Kotlin code? Right now we have this funny-looking clash between Kotlin-style _ and Dolphin-style m.

Reply:
> No, why would it? It's not a member variable.
I think I've been confused with how we've been using the m prefix, woops.

> Also, a broader question: Should we be using the m prefix at all when writing Kotlin code? Right now we have this funny-looking clash between Kotlin-style _ and Dolphin-style m.
I'd be ok with removing it. It has been looking a little silly.